packages/web/src/components/spa/HackAUBG/LandingAnimation/LandingAnimation.jsx

```diff
@@ -1,11 +1,48 @@
 import React from 'react';
+import { useEffect } from 'react';
 import './landing_animation.css';
 
 export const MatrixWindow = () => {
+    const animation = () => {
+        const letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+        let interval = null;
+        const h1 = document.querySelector('.toni-montana');
+
+        let iteration = 0;
+        clearInterval(interval);
+
+        interval = setInterval(() => {
+            h1.innerText = h1.innerText
+                .split('')
+                .map((letter, index) => {
+                    if (index < iteration) {
+                        return h1.dataset.value[index];
+                    }
+
+                    return letters[Math.floor(Math.random() * 26)];
+                })
+                .join('');
+
+            if (iteration >= h1.dataset.value.length) {
+                clearInterval(interval);
+            }
+
+            iteration += 1 / 3;
+        }, 30);
+    };
+
+    useEffect(animation, []);
+
     return (
         <div className="animation-container">
```

Comment:
if you are not using moduled css, you should be more descriptive with your css classes. For example, here the className should probably be "hackaubg-landing-section-text-animation-container"